### DISCUSSION TITLE
Unable to use ACL

### DISCUSSION THREAD

**Discussion by Madhwan-fideo** [drakkan/sftpgo#1546] [Ideas]
I am attempting to configure ACLs on an S3 bucket to manage access on a directory-by-directory basis, but the configurations are not consistently effective. Although I have been using the web UI for this setup, it sometimes works and sometimes does not.
<img width="1371" alt="SCR-20240219-qjko" src="https://github.com/drakkan/sftpgo/assets/149387521/95ec1864-2340-42c7-9ddd-f88b80ea5f3b">

**Comment by codeFideo** [comment]
yeah facing the same issue.
Please help

**Comment by becelot** [comment]
not sure about your exact usecase you want to achieve, but maybe try `/org1` instead of `/org1/*`. Note that these are per-directory permissions, so with `/org1/*` you essentially put the ACL on all subfolders inside `/org1`. If there was a subfolder `/org1/test`, the user would only be allowed to upload in `/org1/test`, but not delete files in this folder, etc.
